src/services/kuma-api/RestClient.spec.ts

```diff
@@ -15,14 +15,8 @@ describe('RestClient', () => {
   })
 
   test.each([
-    ['api', 'http://localhost:5681/api'],
-    ['/api', 'http://localhost:5681/api'],
-    ['/api/', 'http://localhost:5681/api'],
-    ['test/api', 'http://localhost:5681/test/api'],
-    ['/test/api', 'http://localhost:5681/test/api'],
-    ['/test/api/', 'http://localhost:5681/test/api'],
```

Comment:
Why do we remove these test cases? They look like genuine options. (At lest the 2nd and 5th).

Reply:
Because `RestClient` now always expects an absolute URL. The scenario of there only being a root-relative URL is now covered when we read from the Kuma config object.

Reply:
Though it sounds like this test doesn't really make sense anymore (the setter does nothing special)